MarsCompetitionTask/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿using AventStack.ExtentReports;
+using MarsCompetitionTask.Models;
+using MarsCompetitionTask.Pages;
+using MarsCompetitionTask.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsCompetitionTask.Tests
+{
+    [TestFixture]
+    public class CertificationTests:CommonDriver
+    {
+        ProfileHomePage profileHomePageObj;
+        CertificationPage certificationPageObj;
+
+        private static IWebElement popupMsg => driver.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+        private static IWebElement cancelButton => driver.FindElement(By.XPath("//input[@value='Cancel']"));
+        string popUpMsg1 = "has been added to your certification";
+        string popUpMsg2 = "This information is already exist.";
+        string popUpMsg3 = "Duplicated data";
+        string popUpMsg4 = "Please enter Certification Name, Certification From and Certification Year";
+        string popUpMsg5 = "has been updated to your certification";        
+        string popUpMsg6 = "has been deleted from your certification";
+        public CertificationTests()
+        {
+            profileHomePageObj = new ProfileHomePage();
+            certificationPageObj = new CertificationPage();
+        }
+        [Test, Order(1), Description("This test adds a new entry in Certification feature")]
+        public void TestAddCertification()
+        {
+            profileHomePageObj.NavigateToCerticationPanel();
+            certificationPageObj.ClearData();
```

Comment:
this is not done in the correct place, you need to clear all the data before any test runs, there is a hook that you can use to do this! After this the test should delete the test data that it adds please fix this

Reply:
ill let you work out what is the correct hook step to use